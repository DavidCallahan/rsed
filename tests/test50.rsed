list = {} 
list = $list { 0 } { 1 ,2 }
print $list[0]
print $list[2]
print $list[3]
text ="abc"
print $text[0]
print $text[2]
print $text[3]
list = $list { "foo" }
print $list
print join(",", $list)
print join(":",  append({1,2},3,{4,5}))
input $list
foreach all 
   print $CURRENT
end
close input
