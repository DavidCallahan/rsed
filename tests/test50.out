0
2

a
c

012foo
0,1,2,foo
1:2:3:4:5
0
1
2
foo
